backend/controller/user.go

```diff
@@ -167,20 +196,21 @@ func GetUserCreds(w http.ResponseWriter, r *http.Request) {
 
 func DeleteUserCreds(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	err, uId := helpers.ParseInt(w, r, "userId")
+	userIds := struct {
+		Users []uint `json:"users"`
+	}{}
+
+	err := json.NewDecoder(r.Body).Decode(&userIds)
 	if err != nil {
-		return
+		http.Error(w, "Malformed request", http.StatusBadRequest)
 	}
-	res := db.DB.Unscoped().Select(clause.Associations).Where(models.Credential{UserId: uId}).Delete(&[]models.Credential{})
+
+	res := db.DB.Unscoped().Select(clause.Associations).Where("user_id IN ?", userIds.Users).Delete(&[]models.Credential{})
```

Comment:
can u paste new sql output? this looks like it'll perform better tho perhaps

Reply:
It was DELETE * FROM credentials WHRE user_id in ()

Reply:
perfect, ty